server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -144,31 +148,33 @@ export default class BulkChangeModal extends React.PureComponent {
     }
   };
 
-  handleAssigneeSelect = (assignee: string) => {
+  handleAssigneeSelect = (assignee /*: string */) => {
     this.setState({ assignee });
   };
 
-  handleFieldCheck = (field: string) => (checked: boolean) => {
+  handleFieldCheck = (field /*: string */) => (checked /*: boolean */) => {
     if (!checked) {
       this.setState({ [field]: undefined });
     } else if (field === 'notifications') {
       this.setState({ [field]: true });
     }
   };
 
-  handleFieldChange = (field: string) => (event: { target: HTMLInputElement }) => {
+  handleFieldChange = (field /*: string */) => (event /*: { target: HTMLInputElement } */) => {
     this.setState({ [field]: event.target.value });
   };
 
-  handleSelectFieldChange = (field: string) => ({ value }: { value: string }) => {
+  handleSelectFieldChange = (field /*: string */) => ({ value } /*: { value: string } */) => {
     this.setState({ [field]: value });
   };
 
-  handleMultiSelectFieldChange = (field: string) => (options: Array<{ value: string }>) => {
+  handleMultiSelectFieldChange = (field /*: string */) => (
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)